api/controllers/user.py

```diff
@@ -148,17 +148,19 @@ def get_user_profile(user_uuid: str, lang: Language, db: Session) -> s.UserProfi
     )
     locations: list[s.LocationStrings] = [s.LocationStrings(name=name, uuid=uuid) for name, uuid in regions]
 
+    auth_accounts: list[s.AuthAccountOut] = [
+        s.AuthAccountOut(
+            oauth_id=auth_account.oauth_id,
+            email=auth_account.email,
+            auth_type=s.AuthType(auth_account.auth_type),
+        )
+        for auth_account in db_user.auth_accounts
+        if not auth_account.is_deleted
+    ]
+
     return s.UserProfileOut(
-        **pop_keys(db_user.__dict__, ["services", "locations"]),
-        auth_accounts=[
-            s.AuthAccountOut(
-                oauth_id=auth_account.oauth_id,
-                email=auth_account.email,
-                auth_type=s.AuthType(auth_account.auth_type),
-            )
-            for auth_account in db_user.auth_accounts
-            if not auth_account.is_deleted
-        ],
+        **pop_keys(db_user.__dict__, ["services", "locations", "auth_accounts"]),
+        auth_accounts=auth_accounts,
```

Comment:
Why are we replacing this values?

Try using `s.UserProfileOut.model_validate` instead